doc/ref_cert/RC1/chapters/chapter02.rst

```diff
@@ -771,86 +752,95 @@ related to either the VNF domain or the NFVI domain itself:
 The following table contains the list of performance metrics related to
 the VNF domain.
 
-+-------------------+-----------------+-----------+---------------------------+
-| Reference         | Name            | Unit      | Definition/Notes          |
-+===================+=================+===========+===========================+
-| vnf.nfvi.perf.001 | vNIC throughput | bits/s    | Throughput per vNIC       |
-+-------------------+-----------------+-----------+---------------------------+
-| vnf.nfvi.perf.002 | vNIC latency    | second    | Frame transfer time to    |
-|                   |                 |           | vNIC at the throughput    |
-|                   |                 |           | (vnf.nfvi.perf.001)       |
-+-------------------+-----------------+-----------+---------------------------+
-| vnf.nfvi.perf.003 | vNIC delay      | second    | Frame Delay Variation     |
-|                   | variation       |           | (FDV) to vNIC at the      |
-|                   |                 |           | throughput                |
-|                   |                 |           | (vnf.nfvi.perf.001)       |
-+-------------------+-----------------+-----------+---------------------------+
-| vnf.nfvi.perf.004 | vNIC            | number    | Simultaneous active L3/L4 |
-|                   | simultaneous    |           | flows per vNIC before a   |
-|                   | active flows    |           | new flow is dropped       |
-+-------------------+-----------------+-----------+---------------------------+
-| vnf.nfvi.perf.005 | vNIC new flows  | flows/s   | New L3/L4 flows rate per  |
-|                   | rate            |           | vNIC                      |
-+-------------------+-----------------+-----------+---------------------------+
-| vnf.nfvi.perf.006 | Storage         | bytes/s   | Throughput per virtual    |
-|                   | throughput      |           | storage unit              |
-+-------------------+-----------------+-----------+---------------------------+
-| vnf.nfvi.perf.007 | vCPU capacity   | test-\    | Compute capacity per vCPU |
-|                   |                 | specifics |                           |
-|                   |                 | core      |                           |
-+-------------------+-----------------+-----------+---------------------------+
+.. list-table:: Performance metrics related to the VNF domain
+   :widths: 20 20 20 40
+   :header-rows: 1
+
+   * - Reference
+     - Name
+     - Unit
+     - Definition/Notes
+   * - vnf.nfvi.perf.001
```

Comment:
In the RM as well as RA1 we had replaced "nfvi" with "inf", and "perf" with "pm" in the reference